tools/generator/Tests-Core/expected/IntDefinitionAttribute.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+
+namespace Android.Runtime
+{
+	[AttributeUsage (AttributeTargets.Field)]
+	public class IntDefinitionAttribute : Attribute
```

Comment:
This seems odd; why do you need this type in both `expected` and `expected.cp`? The version in `SupportFiles` should be enough, no?